README.rst

```diff
@@ -1,3 +1,23 @@
+.. image:: https://img.shields.io/pypi/v/waitress.svg
+    :target: https://pypi.python.org/pypi/waitress
+    :alt: waitress on PyPI
+
+.. image:: https://img.shields.io/pypi/l/waitress.svg
+    :target: https://pypi.python.org/pypi/waitress
+    :alt: waitress on PyPI
```

Comment:
Let's remove this one. There are other ways to get the license information that are friendly to screen readers.